packages/edit-site/src/hooks/commands/use-common-commands.js

```diff
@@ -176,9 +170,6 @@ function useGlobalStylesOpenCssCommands() {
 	const { openGeneralSidebar, setEditorCanvasContainerView, setCanvasMode } =
 		unlock( useDispatch( editSiteStore ) );
 	const { params } = useLocation();
-	const isMobileViewport = useViewportMatch( 'medium', '<' );
-	const isListPage = getIsListPage( params, isMobileViewport );
```

Comment:
This function was already broken before this PR. I changed the logic a bit to navigate to the "global styles" pages only when no active post/template is selected (no postId in the url)